rdbms/models/database.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from dotenv import load_dotenv
+import os
+
+load_dotenv('./.env')
+
+engine = create_engine(os.environ['DATABASE'], echo=True)
```

Comment:
By convention the environment variable identifier for this should be DATABASE_URL, despite it actually being a URI.